app/models/govbox/message.rb

```diff
@@ -51,7 +52,9 @@ def self.create_message_with_thread!(govbox_message)
   end
 
   def replyable?
-    folder.inbox? && [EGOV_DOCUMENT_CLASS, EGOV_NOTIFICATION_CLASS].include?(payload["class"])
+    folder.inbox? &&
+      [EGOV_DOCUMENT_CLASS, EGOV_NOTIFICATION_CLASS].include?(payload["class"]) &&
```

Comment:
Neposunieme tutu cast do subclass, aby toto bolo ciste od upvs veci?

Reply:
Toto je v GovBox message (z UPVS domeny), tam by to mohlo byt ok, ci?